src/api/store/testimonial.py

```diff
@@ -26,6 +26,8 @@ def get_testimonial_info(self,  context: Context, args) -> Tuple[dict, MassEnerg
       
   def list_testimonials(self, context: Context, args) -> Tuple[list, MassEnergizeAPIError]:
     try:
+      filter_params = get_testimonials_filter_params(context.get_params())
```

Comment:
The testiimonials.list route is only used by the community portal. 
I don't think it should use the filter_params, since none will be passed in.